superset/sql_lab.py

```diff
@@ -50,10 +50,10 @@
 from superset.utils.decorators import stats_timing
 
 config = app.config
-stats_logger = config.get("STATS_LOGGER")
-SQLLAB_TIMEOUT = config.get("SQLLAB_ASYNC_TIME_LIMIT_SEC", 600)
+stats_logger = config["STATS_LOGGER"]
+SQLLAB_TIMEOUT = config["SQLLAB_ASYNC_TIME_LIMIT_SEC"]
 SQLLAB_HARD_TIMEOUT = SQLLAB_TIMEOUT + 60
```

Comment:
Out of scope, but I wonder if `SQLLAB_HARD_TIMEOUT` should be defined in `config.py`, too?

Reply:
I don't think sql lab hard timeout needs to be definable in the config as well, the purpose for it is to provide a hard cutoff in celery if a soft timeout fails (or isn't implemented). The time between soft and hard timeout shouldn't be much, so I don't think allowing configuration gets much here. Happy to reconsider if you think there's a good reason to make it configurable though!

Reply:
Not feeling especially opinionated on this one, but in general I have an aversion to hard coding parameters, especially in the context of distributed/async problems which are extra horrible to debug. So, at the risk of cluttering the config file, I would still vote for introducing the config parameter `SQLLAB_HARD_TIMEOUT_ADDON` or similar with a concise comment might save someone down the line a few hours of grief.

**But**: this ended up sounding much more dramatic than it probably is, so happy either way!